pint/models/spindown.py

```diff
@@ -25,29 +25,28 @@ def __init__(self):
         # The number of terms in the taylor exapansion of spin freq (F0...FN)
         self.num_spin_terms = maxderivs
 
```

Comment:
I see. Well, the code is still being run. And the code fails if I add more F's in the parfile than in the maxderivs limit. Perhaps @luojing1211  can have a brief look at this, as it doesn't seem right.